docker-app/qfieldcloud/subscription/exceptions.py

```diff
@@ -1,6 +1,21 @@
+from rest_framework import status
+from core.exceptions import QFieldCloudException
+
 class SubscriptionException(Exception):
     ...
 
 
-class NotPremiumPlanException(SubscriptionException):
-    ...
+class NotPremiumPlanException(QFieldCloudException):
+    """Raised when TODO ..."""
+
+    # TODO code = ""
+    # message = ""
+    status_code = status.HTTP_403_FORBIDDEN
+
+
+# TODO Rather put here? (from core.exceptions)
+# class SubscriptionInactiveError(QFieldCloudException):
+# 
+# class ReachedMaxOrganizationMembersError(QFieldCloudException):
+# 
+# class QuotaError(QFieldCloudException):
```

Comment:
@suricactus not quite sure what belongs into `subscription.eceptions` and what into `core.exceptions` ?

And should they not all inherit from `QFieldCloudException`?

For example the exceptions (of this PR) raised on Job.create() all kindof relate to Subscription status (not enough space, inactive, no online layers). So should they go into core (together with Job) or into subscription?